### PR TITLE
Add keymap for minivan

### PR DESCRIPTION
**Pull request by marksard**
## Description

Added like_jis keymap for minivan/KUMO keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/thevankeyboards/minivan/keymaps/like_jis/keymap.c [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+#define KC_MLAD  MO(_ADJUST)
+
+#define KC_SCCL  TD(TD_SCCL)
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+  [TD_SCCL] = ACTION_TAP_DANCE_DOUBLE(KC_SCLN, KC_QUOT),
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT( \
```
Comment (fauxpark) [inline keyboards/thevankeyboards/minivan/keymaps/like_jis/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

File: keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+// place overrides here
+#define TAPPING_TERM 200
+#define PREVENT_STUCK_MODIFIERS
```
Comment (drashna) [inline keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h]: This setting should be removed, as it's no longer used.

Specifically, it's the new default behavior for QMK Firmware.  And you can disable it by defining `STRICT_LAYER_RELEASE` instead. 

```suggestion
```

File: keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+#define IGNORE_MOD_TAP_INTERRUPT
+#define TAPPING_TERM_PER_KEY
```
Comment (drashna) [inline keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h]: Woooo 👍

File: keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+   #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+   //#define RGBLIGHT_EFFECT_RGB_TEST
+   //#define RGBLIGHT_EFFECT_ALTERNATING
+#endif
+
+// LED Setting: if you have KUMO you can use RGBLIGHT_ENABLE = yes
+#ifdef RGBLIGHT_ENABLE
+  #define RGB_DI_PIN D0
+  #define RGBLIGHT_TIMER
+  #define ws2812_PORTREG  PORTD
```
Comment (drashna) [inline keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h]: This is not actually needed, nor used.  It should be removed.
```suggestion
```

File: keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  #define ws2812_DDRREG   DDRD
```
Comment (drashna) [inline keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h]: This is not actually needed, nor used.  It should be removed.
```suggestion
```

File: keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h (hunk 3 above)
Comment (drashna) [reply keyboards/thevankeyboards/minivan/keymaps/like_jis/config.h]: (not really an issue, just excited to see) :)

**Comment by drashna** [comment]
Thanks!

**Pull request merged**